### ISSUE TITLE
I'm interested in validation of ocr deid tool

### ISSUE THREAD

**Issue by marcheschi** [pydicom/deid#172]
Hi
I'm interested in validation of ocr deid tool.
How is it possible to contribute?
Thank you
Paolo

**Comment by vsoch** [comment]
Did you mean "our" deid tool, or "OCR" optical character recognition? What tool are you talking about?

**Comment by marcheschi** [comment]
I 'm interested in the OCR cleaning method .
I see in your github deid page that you are looking for collaborators ?
Paolo

**Comment by vsoch** [comment]
This repository? It’s not really related to deid. https://github.com/pydicom/dicom-cleaner. The library itself needs more work and the validation so if you want to take the helm I’d be happy to support.

**Comment by marcheschi** [comment]
Maybe you misunderstood me I do not want the helm, 
I can make tests in order to validate the tool on real world US DICOM images for example.
In my opinione, a tool like this to deidentify  burnt image annotations is very useful.
Paolo

**Comment by vsoch** [comment]
Yes! And I mean to say it would be great if you want to do that.

**Comment by marcheschi** [comment]
Ok then .
How to start? 
Maybe it is better to switch to a gitter chat.
Paolo

**Comment by vsoch** [comment]
You probably need a gold standard set that has different configurations and modalities, and to manually label it and then run the tool to see how well it did.

**Comment by marcheschi** [comment]
Yes this could be a valid approach, to have various configurations applied to different modalities and to different manufacturer.
I think the first step for me is prepare a VM with all necessary tools to make tests.
I installed for this purpose a Solaris vm with  Python 3.8.3 
I installed deid with pip install deid.
Or I have to install a git version in order for you to make modification to the code and for me to apply them to the tests?
